library/src/main/kotlin/it/krzeminski/githubactions/yaml/JobsToYaml.kt

```diff
@@ -68,8 +68,10 @@ private fun Job.toYaml() = buildString {
     append(customArgumentsToYaml())
 }
 
+/* Keep in sync with variable "allRunnerTypes" from "script-generator */
 fun RunnerType.toYaml() =
     when (this) {
+        is RunnerType.Custom -> runsOn
```

Comment:
Could you add a unit test for it?

also, nit:

```suggestion
        is Custom -> runsOn
```

Reply:
done